File: example-databinding/src/main/java/com/xwray/groupie/example/databinding/item/SwipeToDeleteItem.java
package com.xwray.groupie.example.databinding.item;

import androidx.annotation.ColorInt;
import androidx.recyclerview.widget.ItemTouchHelper;

public class SwipeToDeleteItem extends CardItem {
    public SwipeToDeleteItem(@ColorInt int colorRes) {
        super();
    }

    @Override public int getSwipeDirs() {
        return ItemTouchHelper.LEFT | ItemTouchHelper.RIGHT;
    }
}

File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/item/SwipeToDeleteItem.kt
package com.xwray.groupie.example.viewbinding.item

import androidx.annotation.ColorInt
import androidx.recyclerview.widget.ItemTouchHelper

class SwipeToDeleteItem(@ColorInt colorInt: Int) : CardItem(colorInt) {
  override fun getSwipeDirs(): Int = ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT
}
